sonar-wercker.yml

```diff
@@ -0,0 +1,39 @@
+box: checkstyle/sonarqube-maven-git
+
+build:
```

Comment:
we will not use it in wercker for now, as wercker is busy with more critical stuff.
Please remove this file and place it in issue as a comment.
We will use it ones we come close to integration to CI process.